services/notify/src/main.js

```diff
@@ -68,9 +68,10 @@ const load = loader({
   },
 
   db: {
-    requires: ['process', 'cfg'],
+    requires: ['process', 'cfg', 'monitor'],
     setup: ({process, cfg}) => tcdb.setup({
       serviceName: 'notify',
+      monitor: monitor.childMonitor('notify'),
```

Comment:
hmm, I think this should be `notify.childMonitor('db')`. I think I r+'d a patch where we did this with service name in the other services. could we update them all to be like this in this PR?

Reply:
Sounds good. Filed https://github.com/taskcluster/taskcluster/issues/2601 so we don't forget.